source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -88,7 +88,6 @@ Breaking Changes
 .. danger:: Updated ``DifferentialDrive`` and ``MecanumDrive`` classes to use North-West-Up axis conventions to match the rest of WPILib. The Z-axis (i.e. turning) will need to be inverted to restore the old behavior.
 
 - Removed deprecated ``MakeMatrix()`` from ``StateSpaceUtil``
-- Removed deprecated ``CameraServer`` class
```

Comment:
It was the getInstance method of the CameraServer class, not the whole class, but should still be mentioned.

https://github.com/wpilibsuite/allwpilib/pull/4205

Reply:
`getInstance` methods are covered later down in the list